### PR TITLE
Update migrate-dtu-to-vcore.md

### PR DESCRIPTION
**Pull request by ms-amsolo**
Hyperscale specific information added to the "Migrate Geo-replicated databases" section. Table has been updated as discussed

### CODE REVIEW

File: azure-sql/database/migrate-dtu-to-vcore.md [hunk 1]
```diff
@@ -225,7 +225,8 @@ The following table provides guidance for specific migration scenarios:
 |General Purpose|Premium|Upgrade|Must migrate secondary first|
 |Business Critical|General Purpose|Downgrade|Must migrate primary first|
 |General Purpose|Business Critical|Upgrade|Must migrate secondary first|
-
+|Standard|Hyperscale|Lateral|Geo-replication to be turned off before migration to Hyperscale|
```
Comment (blakhani-msft) [inline azure-sql/database/migrate-dtu-to-vcore.md:230]: do we need to add Hyperscale target in the table? Just adding a note on the page about hyperscale should be sufficient.

Comment (blakhani-msft) [reply azure-sql/database/migrate-dtu-to-vcore.md:230]: adding a link to main hyperscale page should be easy to maintain.

[Migrate an existing database to Hyperscale](https://learn.microsoft.com/azure/azure-sql/database/manage-hyperscale-database#migrate-an-existing-database-to-hyperscale)

Comment (ms-amsolo) [reply azure-sql/database/migrate-dtu-to-vcore.md:230]: one liner added along with the Hyperscale doc link

**Comment by rwestMSFT** [comment]
@ms-amsolo @blakhani-msft Please close this and reopen it in `sql-docs-pr`.

**Pull request closed without merging**